### DISCUSSION TITLE
LFRic Documentation and Useful Links

### DISCUSSION THREAD

**Discussion by andrewcoughtrie** [MetOffice/simulation-systems#68] [LFRic]
[LFRic Apps Source Code](https://github.com/metoffice/lfric_apps)
[LFRic Apps Documentation](https://metoffice.github.io/lfric_apps)
[LFRic Core Source Code](https://github.com/metoffice/lfric_core)
[LFRic Core Documentation](https://metoffice.github.io/lfric_core)
[Simulation Systems Working Practices](https://metoffice.github.io/simulation-systems)
[LFRic Fortran Coding Standards]( https://metoffice.github.io/lfric_core/how_to_contribute/index.html#how-to-contribute-index)

Please drop any suggestions for further links as a comment as we can edit this pinned post.

**Comment by BHFock** [comment]
Going forwards, are we expecting the working practices for LFRic to remain separate from the [Simulation Systems Working Practices](https://metoffice.github.io/simulation-systems) or will both be merged?

**Reply by jennyhickson** [reply]
The working practices for LFRic Apps will be included in the Simulation Systems WP's once that repository is finalised.
